hyperdash_cli/constants.py

```diff
@@ -3,9 +3,7 @@
 import six
 
 
-GITHUB_CLIENT_ID = "4793c654216adde7a478"
```

Comment:
Have we discarded this ID?

Reply:
its managed by the server now

Reply:
Yeah, but is it still a usable ID?

Reply:
oh yeah it still works, I'll delete it when spencer updates the prod config